src/components/theme/AppExtras/FavoritesToolbarButton/BasketSearchButton.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { Button } from 'semantic-ui-react';
+import { Icon } from '@plone/volto/components';
+import { Portal } from 'react-portal';
+import BasketSearchPopup from './BasketSearchPopup';
+import basketSVG from '@eeacms/volto-freshwater/icons/basket.svg';
+import './style.less';
+
+const BasketSearchButton = (props) => {
+  const { basket, location } = props;
+
+  const [showMenu, setShowMenu] = useState(false);
+  const menuRef = useRef(null);
+
+  useEffect(() => {
+    const outsideClick = (e) => {
+      if (showMenu && menuRef.current && !menuRef.current.contains(e.target)) {
+        setShowMenu(false);
+      }
+    };
+    document.addEventListener('mousedown', outsideClick);
+    return () => {
+      document.removeEventListener('mousedown', outsideClick);
+    };
+  }, [showMenu]);
+
+  const placement = () => {
+    if (location.pathname === '/data-maps-and-tools/metadata') return true;
+    return false;
+  };
+
+  return (
+    <>
+      {placement() === true && (
+        <Portal
+          node={__CLIENT__ && document.querySelector('.documentFirstHeading')}
+        >
+          <div className="boards-search" ref={menuRef} id="toolbar">
```

Comment:
Don't put in a toolbar, because it breaks the website layout. Move the basket button in the header instead. It should go to top everywhere where we use this feature, not just on the metadata catalogue search page. Also use the existing components for that (BasketToolbarButton, BasketToolbarPopup)